install-browser.mjs

```diff
@@ -0,0 +1,44 @@
+/**
+ * Copyright 2021 Google LLC.
+ * Copyright (c) Microsoft Corporation.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {install, computeExecutablePath} from '@puppeteer/browsers';
+import actions from '@actions/core';
+import fs from 'fs/promises';
+
+try {
+  const version = (await fs.readFile('.browser', 'utf-8')).trim();
+  const cacheDir = process.argv[2] || process.cwd();
+  const browser = version.split('@')[0];
+  const buildId = version.split('@')[1];
+  const result = await install({
+    browser,
+    buildId,
+    cacheDir,
+  });
+
+  actions.setOutput(
+    'executablePath',
+    computeExecutablePath({
+      cacheDir,
+      browser,
+      buildId,
+    })
+  );
+  console.log(result.path);
+} catch (err) {
+  actions.setFailed('Failed to download the browser: ' + err.message);
```

Comment:
Use `` (this wasn't detected by pre-commit because it is only configured to run eslint for `.ts`, not `.mjs`, but we should change that later